lib/qbwc/active_record/job.rb

```diff
@@ -18,7 +20,12 @@ def self.add_job(name, enabled, company, worker_class)
     ar_job.request_index = 0
     ar_job.worker_class = worker_class
     ar_job.save!
-    self.new(name, enabled, company, worker_class)
+
+    jb = self.new(name, enabled, company, worker_class)
+    jb.requests = requests.is_a?(Array) ? requests : [requests] unless requests.nil?
+    jb.data = data unless data.nil?
```

Comment:
Done. IIRC I omitted the extra parameters since they were optional, but I've added them now for cleanliness.